fabric/path.go

```diff
@@ -26,18 +26,22 @@ var (
 	}
 )
 
-// fabricPathToPredicate takes a Fabric path string and transforms it an equivalent Skipper Path* predicate
-func fabricPathToPredicate(fp string) *eskip.Predicate {
-	if fp == "/**" {
+// fabricPathStrToPredicate takes a Fabric path string and transforms it an equivalent Skipper Path* predicate
+func fabricPathStrToPredicate(fps string) *eskip.Predicate {
+	if fps == "/**" {
 		return &eskip.Predicate{Name: predicates.PathSubtreeName, Args: []interface{}{"/"}}
 	}
 
 	for _, repl := range repls {
-		fp = repl.match.ReplaceAllString(fp, repl.repl)
+		fps = repl.match.ReplaceAllString(fps, repl.repl)
 	}
 
 	return &eskip.Predicate{
 		Name: predicates.PathName,
-		Args: []interface{}{fp},
+		Args: []interface{}{fps},
 	}
 }
+
+func applyPath(r *eskip.Route, fp *FabricPath) {
```

Comment:
We could also use path as plain string here because we don't use the object.

Reply:
What do you think @marcinzaremba ?

Sorry comment was not showing before now ...

Reply:
Sure, no problem for me. Can change in the next PR.